processing/src/test/java/io/druid/query/ChainedExecutionQueryRunnerTest.java

```diff
@@ -154,7 +155,7 @@ public void run()
     EasyMock.verify(watcher);
   }
 
-  @Test
+  @Test(timeout = 60000)
```

Comment:
We have seen a lot of concurrency tests fail on slower build machines
because of too short timeouts.
Talking to @cheddar, we agreed setting global timeouts failing the entire
suite would be better in those cases.

On Thu, Jan 8, 2015 at 11:31 AM, Fangjin Yang <notifications@github.com>
wrote:

> In
> processing/src/test/java/io/druid/query/ChainedExecutionQueryRunnerTest.java
> <https://github.com/druid-io/druid/pull/1018#discussion-diff-22675188>:
>
> > @@ -154,7 +155,7 @@ public void run()
> >      EasyMock.verify(watcher);
> >    }
> >
> > -  @Test
> > +  @Test(timeout = 60000)
>
> we can probably specify a shorter timeout
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/druid-io/druid/pull/1018/files#r22675188>.
>